### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by fd0**
What does this PR change? What problem does it solve?
-----------------------------------------------------

Update all dependencies.

### CODE REVIEW

**Comment by MichaelEischer** [comment]
We could probably also upgrade to https://github.com/cenkalti/backoff/releases/tag/v4.0.2 . Minio now also has a v7 release, to which we could switch in some future PR.

File: changelog/unreleased/pull-2927 [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Change: Update dependencies, require Go >= 1.13
```
Comment (MichaelEischer) [inline changelog/unreleased/pull-2927:1]: I think we should merge that with `pull-2600` which updates Go to >= 1.12

Comment (fd0) [reply changelog/unreleased/pull-2927:1]: Ah, okay. i'll have a look

**Comment by fd0** [comment]
> We could probably also upgrade to https://github.com/cenkalti/backoff/releases/tag/v4.0.2 

That doesn't work without changing the code, and I don't have time to look into it right now. We'll leave it at v2 for now.

File: changelog/unreleased/pull-2927 (hunk 1 above)
Comment (fd0) [reply changelog/unreleased/pull-2927:1]: I've merged the changelog files.

File: changelog/unreleased/pull-2600 [hunk 2]
```diff
@@ -1,7 +1,11 @@
-Change: Require Go >= 1.12
+Change: Update dependencies, require Go >= 1.13
 
-Restic now requires Go to be at least 1.12. This allows simplifications in the
+Restic now requires Go to be at least 1.13. This allows simplifications in the
 build process and removing workarounds.
 
+This is also probably the last version of restic still supporting mounting
+repositories via fuse on macOS.
```
Comment (rawtaz) [inline changelog/unreleased/pull-2600:7]: Perhaps add a note about why this is?

Comment (fd0) [reply changelog/unreleased/pull-2600:7]: Good idea

**Pull request merged**